app/validators.py

```diff
@@ -45,3 +46,12 @@ def check_empty_keys(data):
                 return False
 
     return True
+
+
+def verify_satellite_id(id_str):
+    # satellite_id can either be a UUID or a 10 digit number depending on Sat version
+    if verify_uuid_format(id_str):
+        return True
+    elif id_str and re.match(r"^\d{10}$", id_str):
+        return True
+    return False
```

Comment:
```suggestion
def verify_satellite_id(satellite_id):
    return verify_uuid_format(satellite_id) or _verify_legacy_satellite_id(satellite_id)
```

def _verify_legacy_satellite_id(satellite_id):
    return satellite_id and re.fullmatch(r"\d{10}", satellite_id) is not None
```

- _fullmatch_ saves us from _^_ and _$_
- A separate method allows us to use simple boolean expression instead of if/else constructs yet retain readability and semantics.